temporal.py

```
@@ -434,12 +523,19 @@ def get_cache_stats(self):
         }
 
     def clear_cache(self):
-        """Clear the LRU cache."""
+        """
+        Clears the LRU cache.
+        """
         self.memory_db_cache.clear()
         self.logger.info("LRU cache cleared")
 
     def resize_cache(self, new_size):
```

Ruff (0.12.2)

Missing return type annotation for public function `resize_cache`

Add return type annotation: `None`

(ANN201)